cluster/v1/types.go

```diff
@@ -22,11 +22,11 @@ import (
 //
 // The cluster join process follows a double opt-in process:
 //
-// 1. agent on managed cluster creates CSR on hub with cluster UID and agent name.
-// 2. agent on managed cluster creates ManagedCluster on hub.
-// 3. cluster admin on hub approves the CSR for the ManagedCluster's UID and agent name.
-// 4. cluster admin sets spec.acceptClient of ManagedCluster to true.
-// 5. cluster admin on managed cluster creates credential of kubeconfig to hub.
+// 1. Agent on managed cluster creates CSR on hub with cluster UID and agent name.
+// 2. Agent on managed cluster creates ManagedCluster on hub.
+// 3. Cluster admin on hub approves the CSR for the ManagedCluster's UID and agent name.
```

Comment:
Would be a good idea to remove the possessive here. Let's change:
"Cluster admin on hub approves the CSR for the ManagedCluster's UID and agent name." to 
"Cluster admin on hub approves the CSR for the UID and agent name of the ManagedCluster."

Reply:
fixed